src/main/scala/com/github/unchama/seichiassist/listener/PlayerInventoryListener.scala

```diff
@@ -473,6 +473,7 @@ class PlayerInventoryListener(implicit effectEnvironment: EffectEnvironment,
       } else if (itemstackcurrent.getType == Material.BOOK_AND_QUILL) {
         // 投票リンク表示
         player.sendMessage(RED.toString + "" + UNDERLINE + "https://minecraft.jp/servers/54d3529e4ddda180780041a7/vote")
+	player.sendMessage(RED.toString + "" + UNDERLINE + "https://monocraft.net/servers/Cf3BffNIRMERDNbAfWQm")
```

Comment:
インデントは2 spacesを使っています (今回はこちらで修正します)
```suggestion
        player.sendMessage(RED.toString + "" + UNDERLINE + "https://monocraft.net/servers/Cf3BffNIRMERDNbAfWQm")
```